frontend/src/app/modules/grids/widgets/wp-table/wp-table-qs.component.ts

```diff
@@ -1,6 +1,5 @@
 import {Component} from '@angular/core';
 import {AbstractWidgetComponent} from "core-app/modules/grids/widgets/abstract-widget.component";
-import {GridWidgetResource} from "core-app/modules/hal/resources/grid-widget-resource";
 import {WidgetChangeset} from "core-app/modules/grids/widgets/widget-changeset";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/modules/grids/widgets/wp-table/wp-table-qs.component.ts#L4"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>